resources/views/projects/show.twig

```diff
@@ -117,13 +117,14 @@
 
 {% block add_file_panel %}
 <p class="message bg-info text-info">
-Some apps have several catkeys for different parts of the software, like for the main "Application", a "Library" or "Tracker-addon". Here you can add those different sub-projects. Normal, "single-projects", should be named "Application".
+Some apps generate several catkeys files, one for each "module" of the software, like for the "Main app", a "Preference panel", a "Library" or "Tracker-addon" it comes with.<br />
+Here you can add those different translation modules and their catkeys. Normal "single-catkeys" projects should be named "Main app" for consistency.
 </p>
 {{ form_open({url: route('files.store', {project: project.id}), method: POST}) }}
 <div class="input-group">
-    {{ form_text('name', null, {'placeholder': 'Name', 'class': 'form-control'}) }}
+    {{ form_text('name', null, {'placeholder': 'Module name', 'class': 'form-control'}) }}
     <span class="input-group-btn">
-        {{ form_submit('Add', {'class': 'btn btn-default'}) }}
+        {{ form_submit('Add module', {'class': 'btn btn-default'}) }}
```

Comment:
If there is `module` in the placeholder, here it's redundant. They're literally next to each other. Please remove it from the button.